dvc/output/base.py

```diff
@@ -311,6 +325,9 @@ def dumpd(self):
         if self.persist:
             ret[self.PARAM_PERSIST] = self.persist
 
+        if self.store == self.stage.is_repo_import:
```

Comment:
probably
```suggestion
        if self.stage.is_repo_import and self.store:
```
?

Reply:
No, actually that is right (although unclear): the purpose of this is to only add `store: <something>` to the DVC file if it differs from the default. Default is `store: true` for most outputs and `store: false` for repo imports. So this code writes the value of `store` to the DVC file only if 

* it's a repo import and `store: true`
* or it's a normal import and `store: false`